### DISCUSSION TITLE
new package request: rocminfo

### DISCUSSION THREAD

**Discussion by hualet** [linuxdeepin/developer-center#10383] [Developer Discussion | 开发者讨论]
AMD ROCm software is AMD's Open Source stack for GPU computation.

rocminfo is a tool to show system infor mation ofr ROCm.

https://github.com/ROCm/rocminfo

**Comment by hualet** [comment]
@Zeno-sole

**Comment by xzl01** [comment]
mark https://salsa.debian.org/rocm-team

**Comment by Zeno-sole** [comment]
已提交集成 https://github.com/linuxdeepin/developer-center/issues/10509
